ci/pipeline.yml

```diff
@@ -200,8 +200,11 @@ jobs:
       - get: retro
         resource: master
         trigger: true
-      - task: compile
-        file: retro/sbt-buildo/ci/compile.yml
+      - in_parallel:
+        - task: scalafmt
+          file: retro/ci/scalafmt.yml
```

Comment:
instead of adding a scalafmt task to each job, it's probably easier to have a standalone scalafmt job that just checks the whole repo

Reply:
👍 Makes sense, I'll do that